### ISSUE TITLE
Add migration

### ISSUE THREAD

**Issue by cliffckerr** [starsimhub/starsim#493]

**Comment by kaa2102** [comment]
Will migration also encompass household movement, e.g., new births, divorces/splits, aging out, and/or moving, etc.?

**Comment by cliffckerr** [comment]
No, those would be separate. We have a simple version of migration implemented here:
https://github.com/amath-idm/hpvsim/blob/main/hpvsim/people.py#L618
